frontend/packages/employee-frontend/src/state/invoicing-ui.tsx

```diff
@@ -90,6 +93,8 @@ interface InvoiceSearchFilterState {
 interface SharedState {
   units: Result<Unit[]>
   setUnits: Dispatch<SetStateAction<Result<Unit[]>>>
+  employees: Result<Employee[]>
```

Comment:
So, like I commented elsewhere, you could store the options objects here instead of full `Employee` objects.